Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/tutorial_list/TutorialList.kt

```diff
@@ -1064,6 +1065,23 @@ fun createLayoutTutorialList(): List<TutorialSectionModel> {
         tagColor = LayoutListColor
     )
 
+    val tutorial3_7_4 = TutorialSectionModel(
+        title = stringResource(R.string.title_3_7_4),
+        description = "Custom layout to show carousel",
+        action = {
+            Tutorial3_7_4CustomCarousel()
+        },
+        tags = listOf(
+            TAG_COMPOSE,
+            TAG_CUSTOM_LAYOUT,
+            TAG_MEASURABLE,
```

Comment:
There is no measurable and placable in this example. It would be better to add Canvas and gesture tags instead since they are part of the example.